web/models/meeting.ex

```diff
@@ -1,19 +1,54 @@
 defmodule Meetings.Meeting do
-  defstruct title: "", date: DateTime.utc_now(), duration: 0, location: "", description: "", email: "", agenda: ""
+  defstruct title: "", location: "", date: DateTime.utc_now(), duration: 0
 
+  alias Meetings.MeetingType
+  alias Meetings.MeetingDate
```

Comment:
Is there a reason that date is in the struct of Meeting and then also a date in MeetingDate? Seems like that could be redundant?